docs/setup_guides/azure.rst

```diff
@@ -1,267 +1,416 @@
 Installing Pangeo on Azure
---------------------------
+==========================
 
-This guide takes you through the steps necessary to install Pangeo on Microsoft's Azure Cloud Platform.
-We'll make use of Azure's Kubernetes as a Service offering, called AKS (Azure Kubernetes Service),
-for installing Pangeo on Azure.
-Documentation on AKS can be found here: https://docs.microsoft.com/en-gb/azure/aks/.
+In this guide, you'll deploy a Pangeo-style JupyterHub on Microsoft Azure. This will deploy a multi-user JupyterHub enabled with Dask for scalable computing.
 
-.. Note::
-  This guide lays out only the fundamental steps required to install Pangeo on Azure AKS.
-  Further work, for example to secure your cluster, is highly advised but not directly
-  covered here.
+We'll use
 
+* `Azure Kubernetes Service <https://docs.microsoft.com/en-us/azure/aks/intro-kubernetes>`__ (AKS), Azure's managed Kubernetes service
+* The `daskhub Helm Chart <https://github.com/dask/helm-chart/tree/main/daskhub>`__, an easy way to install JupyterHub and Dask-Gateway
 
-Step One: Build Kubernetes service
-==================================
+We describe two deployments, a :ref:`simple` and a :ref:`recommended`. If you're new to Azure, Kubernetes, or JupyterHub, then you should try the simple deployment to verify that the basics work, before moving on to the more advanced recommend deployment.
 
-The first step to installing Pangeo on Azure is to set up a Kubernetes service
-that can be used to run Pangeo. This can be done either by using the web interface
-or by using the Azure commandline interface (CLI). These are both practical options,
-so we'll cover each one in turn.
+This guide uses the Azure CLI to create the Azure Resources.
+At the end, we provide a ``Makefile`` with targets for the AKS cluster and an updated deployment. If you're just looking to deploy a Hub, feel free to use and adapt the Makefiles. If you're looking to build understanding, read through the guide.
+As an alternative to this guide, you might use `Qhub <https://docs.qhub.dev/en/latest/>`_, which provides a higher-level tool to obtain a JupyterHub and Dask deployment on Kubernetes (or HPC).
```

Comment:
I'm still exploring QHub. I ran into a couple issues with my initial attempt on Azure, so I've reverted to just using the Azure CLI and Helm in this guide.